### PR TITLE
About

### LINKED ISSUE #18: About page
build the about container with it's components with the right style, test and i18next
add something challenging  (a map maybe)

### PR DESCRIPTION
**Pull request by olaouso**
Screenshot (253)
Screenshot (254)
Screenshot (255)

about page includes about us three cards :(Our Believes, Idea behind the website, About Us)
then team image
the third part of the page is our team, it includes the team images, names, and titles.
close #18

### CODE REVIEW

File: src/components/AboutUs/index.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  font-family: "Acme";
+  color: #590995;
+}
+.aboutCenter {
+  text-align: center;
+  font-family: "Acme";
+}
+
+.aboutHeader {
+  font-family: "Acme";
```
Comment (Ammar-64) [inline src/components/AboutUs/index.css]: DRY, don’t repeat yourself, the font family is replicated

File: src/components/AboutUs/index.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <div className="aboutHeader">
+        <h2>About Us</h2>
+      </div>
+      <Row>
+        {aboutCardInfo.map((aboutCard, index) => (
+          <AboutCard {...aboutCard} key={index} />
+        ))}
+      </Row>
+      <div className="teamImgDiv">
+        <img src={aboutImg} alt="team img" className="teamImg" />
```
Comment (Ammar-64) [inline src/components/AboutUs/index.jsx]: The img name “aboutImg” and the className “teamImg” are not related here, it’s better to give them related names

File: src/components/OurTeam/index.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.teamMain {
```
Comment (Ammar-64) [inline src/components/OurTeam/index.css]: Whats does team main means, can you give it more meaningful name

File: src/components/TeamMember/index.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  width: 50%;
+  margin-bottom: 1.5em;
+  margin-top: 2em;
+}
+.memberCol {
+  text-align: center;
+  font-family: "Acme";
+  color: #590995;
+}
+.memTitle {
```
Comment (Ammar-64) [inline src/components/TeamMember/index.css]: What is memTitle, can you give more meaningful name

File: src/components/aboutUsCard/index.css [hunk 5]
```diff
@@ -0,0 +1,10 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.cardTitle {
+  color: #590995;
+  margin: 1.8em 0;
+}
+.cardPar {
```
Comment (Ammar-64) [inline src/components/aboutUsCard/index.css]: What is cardPar, please don’t abbreviate and use meaningful names

**Review by Ammar-64 — CHANGES_REQUESTED** [review]
Good job Ola, but can you take into consideration the comments I added

File: src/components/AboutUs/index.css (hunk 1 above)
Comment (olaouso) [reply src/components/AboutUs/index.css]: I removed all the font family in all components and add it to App.css file

File: src/components/OurTeam/index.css (hunk 3 above)
Comment (olaouso) [reply src/components/OurTeam/index.css]: replaced with teamSection

File: src/components/AboutUs/index.jsx (hunk 2 above)
Comment (olaouso) [reply src/components/AboutUs/index.jsx]: I replaced it with aboutTeamImg

File: src/components/TeamMember/index.css (hunk 4 above)
Comment (olaouso) [reply src/components/TeamMember/index.css]: I replaced it with memberTitle

File: src/components/aboutUsCard/index.css (hunk 5 above)
Comment (olaouso) [reply src/components/aboutUsCard/index.css]: I replaced it with cardParagraph

**Review by Ammar-64 — APPROVED** [review]
LGTM

**Review by betulerkoc — APPROVED** [review]

**Pull request merged**